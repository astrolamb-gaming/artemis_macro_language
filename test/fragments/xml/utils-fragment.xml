<mission_data version="1.0">
    <values>
        <value OUTER="Hello, nested" />
        <value TEST="0" />
        <range name="LengthTest">
            <value name="test1" />
            <value name="test2" />
            <value name="test3" />
        </range>
    </values>
    <start>
        <set_variable name="testLength" value="${utils.length('LengthTest')}" />
    </start>

</mission_data>