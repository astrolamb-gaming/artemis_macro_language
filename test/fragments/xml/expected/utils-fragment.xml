<mission_data version="1.0">
    <start>
        <set_variable name="testLength" value="3"/>
    </start>
</mission_data>